.github/workflows/main.yml

```diff
@@ -25,8 +25,8 @@ jobs:
     name: setup
     runs-on: ubuntu-latest
     outputs:
-      sdk-ref: ${{ inputs.sdk-ref || '83740c3595ac16b29896c76a2e8790fba0535ab3' }}
-      package-version: '0.5.0'
+      sdk-ref: ${{ inputs.sdk-ref || '10a4cd62fc39c1e0c56ab2bac3a26210f5515204' }}
+      package-version: '0.5.2'
```

Comment:
Is there a reason why this is not bumped to the latest `0.6.1`?

Reply:
No! It didn't exist yet last week when I created this.